build-source.sh

```diff
@@ -16,5 +16,5 @@ if [ ! -f drugref2.war ]; then
   docker run -v $(pwd):/code/ busybox sh -c "cd /code/ && wget $DRUGREF_WAR -O drugref2.war"
 fi
 
-docker-compose build tomcat_oscar
+./setup-oscar-release.sh
```

Comment:
This isn't right because we don't want the db to be bootstrapped when building source.

Reply:
Right. Should we remove the populate-db in ./setup.sh? I'm thinking I could do this together in #44